### ISSUE TITLE
ImportError: cannot import name 'SortedKeyList'

### ISSUE THREAD

**Issue by turian** [pyannote/pyannote-video#32]
#### Description
`ImportError: cannot import name 'SortedKeyList'` when running the example Jupiter notebook.

#### Steps/Code to Reproduce
I follow the instructions in the README exactly as specified and load the Jupiter notebook, and run everything without modification.

#### Expected Results
The clustering command should terminate successfully.

#### Actual Results

When I get to this line:

```
from pyannote.video.face.clustering import FaceClustering
clustering = FaceClustering(threshold=0.6)
```

I get the following error:

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-10-e6bcbe99db82> in <module>()
----> 1 from pyannote.video.face.clustering import FaceClustering
      2 clustering = FaceClustering(threshold=0.6)

~/anaconda2/envs/pyannote/lib/python3.6/site-packages/pyannote/video/face/clustering.py in <module>()
     33 import itertools
     34 import numpy as np
---> 35 from sortedcollections import ValueSortedDict
     36 
     37 from pandas import read_table

~/anaconda2/envs/pyannote/lib/python3.6/site-packages/sortedcollections/__init__.py in <module>()
     12 )
     13 
---> 14 from .recipes import IndexableDict, IndexableSet
     15 from .recipes import ItemSortedDict, ValueSortedDict
     16 from .recipes import OrderedSet, SegmentList

~/anaconda2/envs/pyannote/lib/python3.6/site-packages/sortedcollections/recipes.py in <module>()
      5 import collections as co
      6 from itertools import count
----> 7 from sortedcontainers import SortedKeyList, SortedDict, SortedSet
      8 from sortedcontainers.sortedlist import recursive_repr
      9 

ImportError: cannot import name 'SortedKeyList'
```

#### Versions
pyannote.algorithms==0.8
pyannote.core==1.3.1
pyannote.parser==0.7.1
pyannote.video==1.6.1

**Comment by turian** [comment]
Can be solved by upgrading sortedcontainers to 2.0.4

**Comment by hbredin** [comment]
Thanks for your feedback! 
Should be solved in latest 1.6.2 release
